README.md

```diff
@@ -322,7 +322,9 @@ To avoid having to run the databse setup for each test run you can specify the `
  which will use an existing test database if one exists:
 
     $ REUSE_DB=1 ./manage.py test corehq.apps.app_manager
-    $ REUSE_DB=reset ./manage.py test corehq.apps.app_manager  # drop the current test DB and create a fresh one
+
+    # drop the current test DB and create a fresh one
+    $ REUSE_DB=1 ./manage.py test corehq.apps.app_manager --reusedb=reset
```

Comment:
I think `REUSE_DB=1` is not necessary if you specify `--reusedb=...`

Reply:
Oh, really? For some reason I thought you needed both. Maybe I'll leave it to you to convince yourself you don't need it and update. (To convince yourself, make sure you don't already have REUSE_DB=1 set in your bash profile!)

Reply:
Sure. Here's the relevant bit of code where the value is retrieved either from the command line OR from the environment if not specified on the command line:

https://github.com/dimagi/commcare-hq/blob/b49a26a4cb40cea352601361c526649c3d6e0ac5/corehq/tests/nose.py#L184-L185